lib/shared/widgets/gradient_background.dart

```diff
@@ -37,7 +37,7 @@ class UniDirectionalBackground extends StatelessWidget {
           gradient: LinearGradient(
         colors: [
           Color(0xFF6CAFB4),
-          Colors.white,
+          Color.fromARGB(255, 223, 242, 187),
```

Comment:
@JayrajSinh16 

**Don't deviate from the pre-determined colorscheme**. If you wish to make changes that are outside the scope of your issue, please discuss it first if what you wish to change conforms to the general flow of the project or not.

**Please revert all the changes made in the `LinearGradient` widget**

Reply:
@RitwikSingh28 
yes sir will revert the changes made in the Linear Gradient widget.